docs/patterns/navigation.md

```diff
@@ -18,8 +18,12 @@ displayed horizontally on larger screens.
   </p>
 </div>
 
-The background color of a navigation pattern can be set via the
-`$color-navigation-background` variable.
+The navigation pattern is on of the firsts to implement the new theming architecture in vanilla. By default, the nav is light. t`o switch to a dark nav, you can either:
+
+- Change the value of the `nav` key in the `$theme-default--dark` map (located in `_settings_themes.scss`) to `true`
+- Add the class `is-dark` when the default nav is light, or `is-light` when the default has been changed to dark
+
+You can also manually override the background color of the nav using the variable `$color-navigation-background`. If the lightness of the background is above 70%, the text colour will switch to dark to improve readibility.
```

Comment:
You do not mention `$color-navigation-text`. Can you also provide examples of there usage for docs and for percy tests.

Reply:
You said this is an internal variable. Its value is set automatically, so not something you can control. (It doesn't have !default on it.)